pylint.cfg

```diff
@@ -1,2 +1,2 @@
 [MESSAGES CONTROL]
-disable=C0103,E1101,I1101,R0801
+disable=C0103,E1101,I1101,R0801,E0606,W0719
```

Comment:
Btw, in principle this goes for the other rules as well. Neither of the listed rules should need to be disabled globally, I think...

Reply:
Should I open an issue for this? Or can I fix it here?

Reply:
Let's just leave the rules that are there and only remove the new ones. I don't think this needs an issue, it's not important enough. Was more of a general comment/advice to think twice when globally disabling linting rules.